R/pkg/DESCRIPTION

```diff
@@ -42,6 +42,7 @@ Collate:
     'context.R'
     'deserialize.R'
     'functions.R'
+    'functions_avro.R'
```

Comment:
I think we can just put into the same `funcitons.R` like we did in https://github.com/apache/spark/pull/29917

Reply:
I thought it might be better to have in a separate script, as it is external after all. 

Not to mention `functions` are already huge and painful to navigate, even without adding new `family` there. Honestly, I'd prefer to split the whole `functions.R` into families (`functions_aggregate.R`, `functions_datettime.R`, ...).

Reply:
Separating them sounds like an idea, yes. But for now I think we should just put it in the same place; otherwise, it would make more sense to place ML ones separately to follow the structure in PySpark side.